LibraryTests/QuantumPhaseEstimationTests.qs

```diff
@@ -0,0 +1,54 @@
+namespace Microsoft.Quantum.Tests {
+
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Canon;
+	open Microsoft.Quantum.Extensions.Convert;
+    open Microsoft.Quantum.Extensions.Testing;
+	open Microsoft.Quantum.Extensions.Math;
+
+
+    /// # Summary
+    /// Assert that the QuantumPhaseEstimation operation for the T gate
+	/// return 0000 in the controlRegister when targetState is 0 and
+	/// return 0010 when the targetState is 1  
+    operation QuantumPhaseEstimation_Test () : ()
```

Comment:
Most of our unit tests have names that follow the CamelCase convention (capitalized, without underscores), following style guide suggestions for operation and function names more generally.